js/repeater-list.js

```diff
@@ -47,7 +47,7 @@
 
 		$.fn.repeater.Constructor.prototype.list_getSelectedItems = function () {
 			var selected = [];
-			this.$canvas.find('.repeater-list table tbody tr.selected').each(function () {
+			this.$canvas.find('.repeater-list table tbody tr.selected, .repeater-list table tbody tr.multi-selected').each(function () {
```

Comment:
We may want to considering just using `selected` instead `multi-selected` in order to simplify now that we have a wrapper class that will cascade.